content/projects/gsoc/gsoc2019-project-ideas.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: project
+title: "GSoC 2019 Project Ideas"
```

Comment:
No need to copy-paste the original page, use the existing one

Reply:
I thnik it should go to projects/gsoc/2019/index.adoc instead of the current redirect